docs/proposals/20201125-jaeger-traces-store.md

```diff
@@ -0,0 +1,122 @@
+# Jaeger traces store
+
+* **Owners:**:
+    * `@pavolloffay`
+
+* **Related Tickets:**
+    * [observatorium/issues/442](https://github.com/observatorium/observatorium/issues/442)
+    * Depends on: [Traces ingestion API and OpenTelemetry collector](https://github.com/observatorium/observatorium/pull/443)
+
+## Table of Contents
+
+- [TLDR](#tldr)
+- [Why](#why)
+- [Goals](#goals)
+- [Non-Goals](#non-goals)
+- [How](#how)
+    * [Jaeger configuration](#jaeger-configuration)
+        + [Single Jaeger instance](#single-jaeger-instance)
+            - [OpenTelemetry collector configuration](#opentelemetry-collector-configuration)
+        + [Jaeger instance per tenant](#jaeger-instance-per-tenant)
+            - [OpenTelemetry collector configuration](#opentelemetry-collector-configuration)
+    * [Jaeger Query](#jaeger-query)
+        + [Expose only Jaeger Query API](#expose-only-jaeger-query-api)
+        + [Expose Jaeger Query UI](#expose-jaeger-ui)
+- [Action Plan](#action-plan)
+- [References](#references)
+
+<small>
+
+<i>
+
+<a href="http://ecotrust-canada.github.io/markdown-toc/">
+Table of contents generated with markdown-toc
+</a>
+
+</i>
+
+</small>
+
+## TLDR
+
+This proposal adds Jaeger deployment with a persistent storage (e.g. Elasticsearch) to store trace data and it as well defines a query API to search for traces.
+
+## Why
+
+Traces store adds a crucial capability to Observatorium to persist and query tracing data.
+
+## Goals
+
+* Configure OpenTelemetry collector to export data to Jaeger collector.
+* Deploy Jaeger with a persistent storage.
+* Define query API for trace data.
+
+## Non-Goals
+
+* Define trace ingestion API - it is done in [Traces ingestion API and OpenTelemetry collector](https://github.com/observatorium/observatorium/pull/443) proposal.
+
+## How
```

Comment:
As @matej-g mentioned, can we form this proposal in a way:

1. Proposed solution
2. Alternatives

Just pick one you believe in the most for now (: Right now it's hard to say which one we propose

Reply:
I am not opposed to restructuring the document but the proposed solution is mentioned in the action plan.